### PR TITLE
Update CSV with new refactoring details. Unresolved rules are marked …

### PR DESCRIPTION
**Pull request by michaelcfanning**
…Port? -> TBD. These should be ignored during auto-generation of code. @genlu @lgolding @srivatsn

### CODE REVIEW

File: RulesInventory.csv [hunk 1]
```diff
@@ -129,32 +129,32 @@ CA1804,RemoveUnusedLocals,Microsoft.Maintainability,#N/A,Remove unused locals,Un
...
-RS0011,#N/A,NotCategorized,Roslyn.Diagnostics.Analyzers,CancellationToken parameters must come last,#N/A,,,,,,ApiDesign,,,
-RS0012,#N/A,NotCategorized,Roslyn.Diagnostics.Analyzers,Do not call ToImmutableArray on an ImmutableArray value,#N/A,,,,,,Reliability,,,
+RS0006,#N/A,Microsoft.Composition,Roslyn.Diagnostics.Analyzers,Do not mix attributes from different versions of MEF,#N/A,,,,,,Reliability,,,
+RS0007,#N/A,NotCategorized,Roslyn.Diagnostics.Analyzers,Avoid zero-length array allocations.,#N/A,,,,,,Performance,,TBD,
```
Comment (srivatsn) [inline RulesInventory.csv]: This would be in System.Runtime

File: RulesInventory.csv [hunk 2]
```diff
@@ -129,32 +129,32 @@ CA1804,RemoveUnusedLocals,Microsoft.Maintainability,#N/A,Remove unused locals,Un
...
+CA2001,AvoidCallingProblematicMethods,Api.Review,#N/A,Avoid calling problematic methods,A member calls a potentially dangerous or problematic method.,Yes,279199,1198,0.004290846,1269.193408,Reliability,High,TBD,
+CA2104,DoNotDeclareReadOnlyMutableReferenceTypes,System.Runtime,#N/A,Do not declare read only mutable reference types,An externally visible type contains an externally visible read-only field that is a mutable reference type. A mutable type is a type whose instance data can be modified.,Yes,292444,8727,0.029841611,183.1684881,Security,Low,Cut,
 RS0001,#N/A,Roslyn.Diagnostics.Analyzers,Roslyn.Diagnostics.Analyzers,Use SpecializedCollections.EmptyEnumerable(),#N/A,,,,,,Performance,,,
 RS0002,#N/A,Roslyn.Diagnostics.Analyzers,Roslyn.Diagnostics.Analyzers,Use SpecializedCollections.SingletonEnumerable(),#N/A,,,,,,Performance,,,
-RS0003,#N/A,NotCategorized,Roslyn.Diagnostics.Analyzers,Do not directly await a Task,#N/A,,,,,,Reliability,,,
+RS0003,#N/A,NotCategorized,Roslyn.Diagnostics.Analyzers,Do not directly await a Task,#N/A,,,,,,Reliability,,TBD,
```
Comment (srivatsn) [inline RulesInventory.csv]: This would probably be in System.Threading.Tasks

**Comment by genlu** [comment]
@michaelcfanning Could you please make the change suggested by @srivatsn and merge this?

**Comment by genlu** [comment]
Close this since the change was reapplied in https://github.com/dotnet/roslyn-analyzers/pull/325 and https://github.com/dotnet/roslyn-analyzers/pull/327

**Pull request closed without merging**